modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -547,6 +576,17 @@ ONNXCompiled::ONNXCompiled(const gapi::onnx::detail::ParamDesc &pp)
         }
     }
 
+    if (!params.const_inputs.empty()) {
+        // Form input names order without const input names
+        in_names_without_const.clear();
```

Comment:
Is this vector left empty in the case when there's no `constInput`-s specified?

Reply:
Yes. It was invented for const input. `clear` just in case.